packages/integration-tests/api/api.js

```diff
@@ -394,6 +394,121 @@ async function addProviderApi({ prefix, provider }) {
   });
 }
 
+/**
+ * Fetch a list of providers from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @returns {Promise<Object>} - the list of providers fetched by the API
+ */
+async function getProviders({ prefix }) {
+  const payload = await callCumulusApi({
```

Comment:
This assignment is not necessary.  Could just be `return callCumulusApi({`

Reply:
changed